pkg/client/v1/model/active_directory_membership.go

```diff
@@ -3,29 +3,77 @@
  */
 
 package model
-//package nimblestorage/v1/ActiveDirectoryMembership
 
 
-// ActiveDirectoryMembership :
+
+// ActiveDirectoryMembership - Manages the storage array's membership with the Active Directory.
+// Export ActiveDirectoryMembershipFields for advance operations like search filter etc.
+var ActiveDirectoryMembershipFields *ActiveDirectoryMembership
+
+func init(){
+	IDfield:= "id"
+	Descriptionfield:= "description"
+	Namefield:= "name"
+	Netbiosfield:= "netbios"
+	ServerListfield:= "server_list"
+	ComputerNamefield:= "computer_name"
+	OrganizationalUnitfield:= "organizational_unit"
+	Userfield:= "user"
+	Passwordfield:= "password"
```

Comment:
Why do we need to declare these here as <parameterName>field when we simply use them down below in the <Object>Fields initializer?

Just set the value directly on the declaration below and we eliminate the need for all these extra lines.

Reply:
Object members are pointers.

Reply:
Ya.  That's right.  And there's no clean one-liner to do this in golang.  Oh well.  Thanks.